### ISSUE TITLE
Update readme

### ISSUE THREAD

**Issue by Karan-Mahato** [iamparas0/TIC-TAC-TOE#137]
Add contributer section at the end of the readme section to show whim have contributed to this project.

Assign this issue to me, I would like to work on this project.

Also assign labels to this issue.

**Comment by iamparas0** [comment]
> Add contributer section at the end of the readme section to show whim have contributed to this project.
> 
> Assign this issue to me, I would like to work on this project.
> 
> Also assign labels to this issue.

Go for it, please attach ss before pr don't raise direct pr .

##note : the section must be minimal and clean don't add anything extras , keep it as simple as you can

**Comment by Karan-Mahato** [comment]
![image](https://github.com/user-attachments/assets/ac8ff285-488c-41ff-aaa7-f57579cabca3)
Here is the updated README image. Should I merge the feature branch and create a pull request, or do you want any changes?

**Comment by iamparas0** [comment]
> ![image](https://private-user-images.githubusercontent.com/141478419/374443281-ac8ff285-488c-41ff-aaa7-f57579cabca3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgzNzk3NjcsIm5iZiI6MTcyODM3OTQ2NywicGF0aCI6Ii8xNDE0Nzg0MTkvMzc0NDQzMjgxLWFjOGZmMjg1LTQ4OGMtNDFmZi1hYWE3LWY1NzU3OWNhYmNhMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAwOFQwOTI0MjdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zMzcyY2ZmOTY0MzJkZTcwYjMzMWQ5YWVjYmMzOWI4MTAzNzhkNzdkYzAzNjljN2MxMWNjMzM4NDY1MzNlOGFkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.7Ez50kKPtWZBuYwwxdTxV3qoYPs2G407MWWoNADMhDI) ![image](https://private-user-images.githubusercontent.com/141478419/374446277-2d6e38d8-613c-44f9-80b9-9bf77d53210a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgzNzk3NjcsIm5iZiI6MTcyODM3OTQ2NywicGF0aCI6Ii8xNDE0Nzg0MTkvMzc0NDQ2Mjc3LTJkNmUzOGQ4LTYxM2MtNDRmOS04MGI5LTliZjc3ZDUzMjEwYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAwOFQwOTI0MjdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05MDJlYWQzZjNmYTI0N2ZjYjA4NzAwMjRiYmRmNmRkMjEyNzRmMTE3ODhhNDlkMDVlZTEzODg5OGVmNWQ3MWZlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9._GGDYNRoPj8SvwtoxmIJ5pVDtd9McP7EZ1c2sezvB1g)
> 
> @iamparas0 Here is the updated README image. Should I merge the feature branch and create a pull request, or do you want any changes?

is this auto add new contributors also?

**Comment by Karan-Mahato** [comment]
@iamparas0 Yes, contrib.rocks auto update contributors but it take atleast 24hrs to update.
So, I am trying to use all-contributors-cli
